Source/Magick.NET.CrossPlatform/ubuntu.16.04/Build.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
```

Comment:
Can you revert, move and change this script? And can you keep the Docker file inside the CrossPlatform folder? I rather not have these files in the ImageMagick folder. I can also do merge this as is and do that myself. But if you have time to make these changes you would make me a happy man :)

Reply:
Unfortunately that's the best option I could come up with. The dockerfile cannot refer to anything outside it's home folder or subfolders (the build context), so for it to include the ImageMagick checkouts it must be in that folder or a parent. I could go back to having docker check everything out itself, but I really think this setup makes more sense. It certainly was easier to work with when, for example, I needed to fork `PNG`.

Reply:
I'll say it does simplify the build as well; no need to have a `--no-cache` option when there's no cached checkout layer to worry about, and the .dockerignore keeps the build context pretty small and fast. Saves a lot of rebuilding/reinstalling time if you find yourself iterating on something.

Reply:
Maybe you can use this trick: https://stackoverflow.com/a/34300129?

Reply:
Even with the `-f` option, the dockerfile still has to be within the build context (the ImageMagick/Source folder or subfolders). The .dockerignore would also have to stay where it is (or let the build context include everything, which slows it down A LOT).

Reply:
I could move it (and the "verify" dockferfile) into an `/ImageMagick/Dockerfiles` folder and condense the .dockerignores to a single file at '/ImageMagick/.dockerignore` if that would make it seem tidier?

Reply:
Like this:
https://github.com/superstator/Magick.NET/tree/dockerfolders/ImageMagick

I'd still need to go back and update all the pathing in the actual dockerfiles, but that could work.